core/src/main/java/org/opensearch/sql/calcite/udf/mathUDF/ConvFunction.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.sql.calcite.udf.mathUDF;
+
+import java.math.BigInteger;
+import org.opensearch.sql.calcite.udf.UserDefinedFunction;
+
+public class ConvFunction implements UserDefinedFunction {
+  @Override
+  public Object eval(Object... args) {
+    Object number = args[0];
```

Comment:
why not check args.length in ConvFunction? and check args.length in ModFunction.

Reply:
And need some UT or IT for edge case